src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -51,34 +67,87 @@ public void initializeConverter() {
 
         clusterColorControl = (ZclColorControlCluster) endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID);
         if (clusterColorControl == null) {
-            logger.error("Error opening device control controls {}", endpoint.getIeeeAddress());
+            logger.error("Error opening device color controls {}", endpoint.getIeeeAddress());
```

Comment:
Will change it to standard logging (this was part of the original implemetation anyway).

BTW, i think this should be an error. When we arrive at this point of code, the channel is created because it detected color is supported, and here it happens we cannot get the cluster. This should never happen, and if it does the channel will not work. Error or Warn ? (I do not think it should be debug here)

Reply:
Yes - I know some logging in the current code doesn't confirm to the "standard" - I've changed most of it in my next PR, but would like to standardise this as much as possible as it really makes things easier for debugging (lesson learnt from the ZWave binding).

Agreed - error is fine here.